src/jobs/optimize-image.ts

```diff
@@ -148,6 +150,17 @@ const optimizeImage = async (job: Job) => {
         data: { url: urls },
       });
     } else if (model === "artistAvatar") {
+      // TODO: where should the favicon be stored
```

Comment:
We can store it on minio, for which there's a utility library in `minio.ts`. 

You can see an example of it being used here https://github.com/funmusicplace/mirlo/pull/912/files#diff-115f11f048b07ecdfa25349373f3479306ff732a6f8185f5590f98f114d211dbR114